src/server/node_manager.c

```diff
@@ -4985,7 +4986,7 @@ is_request(int stream, int version)
 							}
 						}
 					}
-					propagate_socket_licensing(pmom);
+					propagate_socket_licensing(pmom, 1);
```

Comment:
It looks like the definition of this function is not updated with the new parameter introduced. Can you please take care of it and also declare this in the corresponding header file.